### PR TITLE
RRTMGP updates and bugfixes

### PR DESCRIPTION
**Pull request by dustinswales**
# PR Checklist

- [X] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [X] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [X] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [X] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Description
This PR contains changes to RRTMGP in FV3/ccpp-physics. No changes are needed above fv3atm.

## Testing
These changes were tested on Hera using the Intel compilers.
Not all of the errors addressed here were uncovered by the standard RTs, but rather showed up when running in the EMC workflow. I was able to replicate these errors by extending the RTs and running the tests for longer forecast times.

Updates to the RT baselines are needed, as well as to the input data.

**Input data updated**

- [x] hera.{intel,gnu}
- [x] orion.intel
- [x] cheyenne.{intel,gnu}
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3
- [ ] CI (container) - Not necessary
- [x] S3

**Full regression tests on all tier-1 platforms**

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [n/a] wcoss_dell_p3 - skip this time due to ongoing network hardware issues
- [x] CI: https://github.com/ufs-community/ufs-weather-model/actions/runs/1313432297

## Dependencies
- waiting on https://github.com/NOAA-EMC/fv3atm/pull/392
- waiting on https://github.com/NCAR/ccpp-physics/pull/733

### CODE REVIEW

**Review by climbfuji — CHANGES_REQUESTED** [review]
CICE and MOM6 submodule pointers need updating.

File: tests/parm/control_rrtmgp.nml.IN [hunk 1]
```diff
@@ -180,11 +180,11 @@ deflate_level=1
   do_RRTMGP      = .true.
   active_gases   = 'h2o_co2_o3_n2o_ch4_o2'
   ngases         = 6
-  lw_file_gas        = 'rrtmgp-lw-prototype-g128-210413.nc'
+  lw_file_gas        = 'rrtmgp-data-lw-g128-210809.nc'
```
Comment (climbfuji) [inline tests/parm/control_rrtmgp.nml.IN:183]: @dustinswales Do we need to add these files to our input data directories? Or are they already present?

**Review by climbfuji — COMMENTED** [review]
Please clarify if additional input data is needed, and which regression tests will change results.

File: tests/parm/control_rrtmgp.nml.IN (hunk 1 above)
Comment (dustinswales) [reply tests/parm/control_rrtmgp.nml.IN:183]: @climbfuji Yes. There 2 new files, in rte-rrtmgp/rrtmgp/data, need to go into the input data directory.

File: tests/rt.sh [hunk 2]
```diff
@@ -242,13 +242,14 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   module load ecflow
   ECFLOW_START=ecflow_start.sh
 
-  QUEUE=batch
-  COMPILE_QUEUE=batch
+  QUEUE=debug
```
Comment (climbfuji) [inline tests/rt.sh]: You need to revert lines 245, 246, 248 before I can start regresssion testing, please.

Comment (climbfuji) [reply tests/rt.sh]: And line 252. The only difference in `rt.sh` can be line 448 (`BL_DATE`).

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: hera
Compiler: gnu
Job: BL
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/739437060/20211006193015/ufs-weather-model
Please make changes and add the following label back:
hera-gnu-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: gaea
Compiler: intel
Job: BL
Repo location: /lustre/f2/pdata/ncep/emc.nemspara/autort/pr/739437060/20211006193006/ufs-weather-model
Please manually delete: /lustre/f2/scratch/emc.nemspara/FV3_RT/rt_3807
Test compile 004 failed failed 
Test compile 001 failed failed 
Test compile 002 failed failed 
Test compile 006 failed failed 
Test compile 003 failed failed 
Test compile 005 failed failed 
Test compile 007 failed failed 
Please make changes and add the following label back:
gaea-intel-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: orion
Compiler: intel
Job: BL
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/739437060/20211006143010/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_355684
Test compile 001 failed failed 
Test compile 006 failed failed 
Test compile 004 failed failed 
Test compile 002 failed failed 
Test compile 005 failed failed 
Test compile 003 failed failed 
Test compile 007 failed failed 
Please make changes and add the following label back:
orion-intel-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: jet
Compiler: intel
Job: BL
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/739437060/20211006193008/ufs-weather-model
Please manually delete: /lfs4/HFIP/h-nems/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_30997
Test compile 005 failed failed 
Test compile 002 failed failed 
Test compile 001 failed failed 
Test compile 004 failed failed 
Test compile 003 failed failed 
Test compile 007 failed failed 
Test compile 006 failed failed 
Please make changes and add the following label back:
jet-intel-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: hera
Compiler: intel
Job: BL
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/739437060/20211006201511/ufs-weather-model
Please manually delete: /scratch1/NCEPDEV/stmp2/emc.nemspara/FV3_RT/rt_20820
Test control_c192 011 failed failed 
Test control_c192 011 failed in run_test failed 
Please make changes and add the following label back:
hera-intel-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: hera
Compiler: intel
Job: BL
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/739437060/20211006193729/ufs-weather-model
Please manually delete: /scratch1/NCEPDEV/stmp2/emc.nemspara/FV3_RT/rt_10080
Baseline creation and move successful
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/739437060/20211006205422/ufs-weather-model
Please manually delete: /scratch1/NCEPDEV/stmp2/emc.nemspara/FV3_RT/rt_11208
Test cpld_control_p7 001 failed in check_result failed 
Test cpld_control_p7 001 failed in run_test failed 
Test cpld_debug_p7 011 failed in check_result failed 
Test cpld_debug_p7 011 failed in run_test failed 
Test cpld_decomp_p7 004 failed in check_result failed 
Test cpld_decomp_p7 004 failed in run_test failed 
Test control_c48 019 failed in check_result failed 
Test control_c48 019 failed in run_test failed 
Test regional_control 030 failed in check_result failed 
Test regional_control 030 failed in run_test failed 
Test regional_quilt 032 failed in check_result failed 
Test regional_quilt 032 failed in run_test failed 
Test cpld_control_wave_p7 005 failed in check_result failed 
Test cpld_control_wave_p7 005 failed in run_test failed 
Test regional_quilt_netcdf_parallel 035 failed in check_result failed 
Test regional_quilt_netcdf_parallel 035 failed in run_test failed 
Test regional_quilt_hafs 034 failed in check_result failed 
Test regional_quilt_hafs 034 failed in run_test failed 
Test control_c192 020 failed in check_result failed 
Test control_c192 020 failed in run_test failed 
Test regional_quilt_RRTMGP 036 failed in check_result failed 
Test regional_quilt_RRTMGP 036 failed in run_test failed 
Test cpld_2threads_p7 003 failed in check_result failed 
Test cpld_2threads_p7 003 failed in run_test failed 
Test control_rrtmgp 042 failed in check_result failed 
Test control_rrtmgp 042 failed in run_test failed 
Test regional_quilt_2threads 033 failed in check_result failed 
Test regional_quilt_2threads 033 failed in run_test failed 
Test control_csawmg 044 failed in check_result failed 
Test control_csawmg 044 failed in run_test failed 
Test control_ras 047 failed in check_result failed 
Test control_ras 047 failed in run_test failed 
Test control_flake 046 failed in check_result failed 
Test control_flake 046 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson 050 failed in check_result failed 
Test fv3_HAFS_v0_hwrf_thompson 050 failed in run_test failed 
Test control_thompson 048 failed in check_result failed 
Test control_thompson 048 failed in run_test failed 
Test control_thompson_no_aero 049 failed in check_result failed 
Test control_thompson_no_aero 049 failed in run_test failed 
Test control_csawmgt 045 failed in check_result failed 
Test control_csawmgt 045 failed in run_test failed 
Test control_c384 021 failed in check_result failed 
Test control_c384 021 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 051 failed in check_result failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 051 failed in run_test failed 
Test hafs_regional_atm 075 failed in check_result failed 
Test hafs_regional_atm 075 failed in run_test failed 
Test control_rrtmgp_c192 043 failed in check_result failed 
Test control_rrtmgp_c192 043 failed in run_test failed 
Test control_atm_aerosols 092 failed in check_result failed 
Test control_atm_aerosols 092 failed in run_test failed 
Test datm_cdeps_mx025_gefs 087 failed in check_result failed 
Test datm_cdeps_mx025_gefs 087 failed in run_test failed 
Test hafs_regional_docn 077 failed in check_result failed 
Test hafs_regional_docn 077 failed in run_test failed 
Test hafs_regional_atm_ocn 076 failed in check_result failed 
Test hafs_regional_atm_ocn 076 failed in run_test failed 
Test hafs_regional_docn_oisst 078 failed in check_result failed 
Test hafs_regional_docn_oisst 078 failed in run_test failed 
Test control_c384gdas 022 failed in check_result failed 
Test control_c384gdas 022 failed in run_test failed 
Test cpld_control_c192_p7 006 failed in check_result failed 
Test cpld_control_c192_p7 006 failed in run_test failed 
Test cpld_bmark_p7 010 failed in check_result failed 
Test cpld_bmark_p7 010 failed in run_test failed 
Test cpld_control_c384_p7 008 failed in check_result failed 
Test cpld_control_c384_p7 008 failed in run_test failed 
Test control_decomp 013 failed failed 
Test control_decomp 013 failed in run_test failed 
Please make changes and add the following label back:
hera-intel-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: orion
Compiler: intel
Job: BL
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/739437060/20211006150016/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_209948
Baseline creation and move successful
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/739437060/20211006161653/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_94577
Test control_c384gdas 022 failed failed 
Test control_c384gdas 022 failed in run_test failed 
Please make changes and add the following label back:
orion-intel-BL

**Review by MinsukJi-NOAA — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]

**Review by DusanJovic-NOAA — APPROVED** [review]

**Comment by climbfuji** [comment]
@dustinswales fv3atm was merged, new hash is f7ed0e0.

**Review by climbfuji — APPROVED** [review]

**Pull request merged**